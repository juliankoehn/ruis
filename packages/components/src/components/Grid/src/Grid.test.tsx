import React from 'react';
import { render } from '@testing-library/react';
import Grid from './Grid';

describe('Grid', () => {
  it('Should render a container', () => {
    let title = 'Test';
    const { container } = render(<Grid>{title}</Grid>);

    expect(container).toMatchSnapshot();
  });
});
