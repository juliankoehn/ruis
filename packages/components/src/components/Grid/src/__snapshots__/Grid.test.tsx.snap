// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Should render a container 1`] = `
<div>
  <div
    class="css-1lvpr75-GridStyles-fluidStyles"
  >
    Test
  </div>
</div>
`;
